core/src/org/apache/cloudstack/storage/command/AttachAnswer.java

```diff
@@ -34,6 +37,12 @@ public AttachAnswer(DiskTO disk) {
         setDisk(disk);
     }
 
+    public AttachAnswer(DiskTO disk, Map<String, String> diskDetails) {
```

Comment:
@bhaisaab Thanks for fixing the access level. The code seems ok, just one more question.

Sorry to ask you now, I should have seen this before. Is this constructor used (**AttachAnswer(DiskTO disk, Map<String, String> diskDetails)**)?

Thanks.